lib/gooddata/lcm/types/scalar/integer.rb

```diff
@@ -13,8 +13,8 @@ class IntegerType < BaseType
         CATEGORY = :scalar
         INTERNAL_TYPE = Integer
 
-        def check(_value)
-          true
+        def check(value)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. You can also write as `check(*)` if you want the method to accept any arguments but don't care about them.